arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -5897,6 +5907,14 @@
 				minimumVersion = 100.13.0;
 			};
 		};
+		00BE7B36275AEFD60012CE95 /* XCRemoteSwiftPackageReference "firebase-ios-sdk" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/firebase/firebase-ios-sdk";
+			requirement = {
+				kind = upToNextMajorVersion;
+				minimumVersion = 8.10.0;
+			};
+		};
```

Comment:
I've never used Firebase before. What impact does adding this dependency have on app size (how large is the Firebase framework)? Also, will we need to change the App Privacy section of our App Store page?

For what it is worth, I generally am not in favor of adding third part dependencies unless there is an extremely compelling reason to do so. Each dependency, especially one like this that executes code even before that app is useable by the user, is a failure point. Remember a while back when a bunch of prominent apps stopped working for a while because they used a popular third party analytics framework that had a problem? That is the power that dependencies like this hold. In my opinion, it isn't worth it. But that's up to you and the samples team to decide.

Reply:
- app size
The impact is quite minimum. The previous version (100.13.0.1) of sample viewer app is ~85 MB, and the current version (100.13.0.2) is 87 MB, according to App Store on iPhone 11.
- App Privacy section
I've included usage data (select sample, category, and source code view) and search history (search bar) policies, prior to the release.
- third party dependencies
I agree. I also tried to avoid using 3rd party dependencies in my personal project.  
We researched a few options: Google Analytics, Amazon Pinpoint, Adobe Analytics, and building an AWS based in house solution. Eventually Trevor decided GA is the easiest way to go for now, as we wanted to use it as an experiment to see what data we can get. In the future we may decide to move to other solutions or remove GA dependency.